project/Dependencies.scala

```diff
@@ -104,6 +103,13 @@ object Dependencies {
     ExclusionRule("org.apache.hadoop")
   ) excludeAll(parquetList:_*)
 
+  def sparkCSV: Seq[ModuleID] = {
+    import scala.math.Ordering.Implicits._
+    if (sparkVersionTuple >= (1, 3, 0)) {
```

Comment:
maybe this check isn't worth, do we care about spark older than `1.3` or `1.4`?

looks like `df.write.format("xyz")` was introduced in spark 1.4, so this line [might not compile against older spark](https://github.com/andypetrella/spark-notebook/pull/411/files#diff-a35f3ca29824ef65c8c93013b8233ec9R43).